dbus/methods_test.go

```diff
@@ -507,3 +507,56 @@ func TestConnJobListener(t *testing.T) {
 		t.Fatal("JobListener jobs leaked")
 	}
 }
+
+// Enables a unit and then masks/unmasks it
+func TestMaskUnmask(t *testing.T) {
+	target := "mask-unmask.service"
+	conn := setupConn(t)
+
+	setupUnit(target, conn, t)
+	abs := findFixture(target, t)
+	runPath := filepath.Join("/run/systemd/system/", target)
+
+	// 1. Enable the unit
+	install, changes, err := conn.EnableUnitFiles([]string{abs}, true, true)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if install != false {
+		t.Log("Install was true")
```

Comment:
For reference, this is due to a semantic break after https://github.com/systemd/systemd/pull/3016